packages/react-native/src/Authenticator/common/DefaultContainer/InnerContainer.tsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import { View } from 'react-native';
 
-import { innerContainerstyles } from './styles';
+import { useTheme } from '../../../theme';
+import { getInnerContainerStyles } from './styles';
 import { ContainerProps } from './types';
 
 export default function InnerContainer({
   children,
   ...rest
 }: ContainerProps): JSX.Element | null {
+  const theme = useTheme();
+
+  const themedStyle = getInnerContainerStyles(theme);
```

Comment:
This should be wrapped in a `useMemo` but it needs to move to `DefaultContainer` so will handle it then